python/acl_anthology/anthology.py

```diff
@@ -137,9 +137,15 @@ def load_all(self) -> Self:
             iterator = track(
                 it.chain(
                     self.collections.values(),
-                    (self.people, self.events, self.sigs, self.venues),
+                    (
+                        self.collections.bibkeys,
+                        self.people,
+                        self.events,
+                        self.sigs,
+                        self.venues,
+                    ),
                 ),
-                total=len(self.collections) + 4,
+                total=len(self.collections) + 5,
```

Comment:
Can you add a comment explaining the magic constant?

Reply:
Refactored now to avoid